src/test/lib/sql/sql_translator_test.cpp

```diff
@@ -378,7 +378,7 @@ TEST_F(SQLTranslatorTest, SelectListAliasUsedInOrderBy) {
       sql_to_lqp_helper("SELECT a AS x, b AS y FROM int_float ORDER BY x, y");
 
   const auto aliases = std::vector<std::string>({"x", "y"});
-  const auto sort_modes = std::vector<SortMode>({SortMode::Ascending, SortMode::Ascending});
+  const auto sort_modes = std::vector<SortMode>({SortMode::AscendingNullsFirst, SortMode::AscendingNullsFirst});
```

Comment:
General question: I wonder where we need to add tests for `NULLS LAST`? Getting this PR landed helps a lot, but I don't know when we will revisit important parts where we do not properly test?

Reply:
Checked on `AggregateSort`, that seems to be fine and added some SQLTranslator tests. Also added a NULLs last sort mode in tests for LQPTranslator. I guess that's now pretty much everywhere where we currently have to take care.

The indidually sorted by thing: What do you think? We either have to change the operators or the method description.

Reply:
Added a test for that